postinstall.js

```diff
@@ -0,0 +1,6 @@
+const fs = require('fs');
```

Comment:
Theres no extra module needed in theory. With this small script it works just fine.
Im still not 100% sure if thats what we want.

Reply:
The tests now passed, but thers an linting error. Ill fix this tomorrow. Lets review it tomorrow as well!

Reply:
Not sure if another script is more confusing? What command did you use with that npm package? I think for the last param you cold use a filename or a `.`?

Reply:
I used `copyfiles .env.example .env` 😅 I'll test it tomorrow because I already turned off my computer for today 😅

Reply:
>  I'll test it tomorrow because I already turned off my computer for today

yes it is late, no rush, get some rest. Thank you